### PR TITLE
Implement cookie timeout

### PR DESCRIPTION
**Pull request by tomling**
- Updated to use version 4.24.0 of Microsoft.Identity.Client.
- Implemented version 1.4.1 of Microsoft.Identity.Web.UI (Replacing Microsoft.AspNetCore.Authentication.AzureAD.UI).
- Configured CookieAuthenticationOptions to implement a 1 hour cookie timeout with sliding expiration.

### CODE REVIEW

**Comment by tomling** [comment]
Do we still need CookiePolicyOptions line 113 Startup.cs?

**Review by SamirHassoun — APPROVED** [review]

File: src/NCNEPortal/Startup.cs [hunk 1]
```diff
@@ -140,6 +126,25 @@ public void ConfigureServices(IServiceCollection services)
                 options.UseOracle(hpdConnection));
 
             services.AddSingleton<AppVersionInfo>();
+
+            services.AddMicrosoftIdentityWebAppAuthentication(Configuration, "NCNEPortalAzureAd");
+
+            services.AddRazorPages().AddRazorRuntimeCompilation().AddMvcOptions(options =>
```
Comment (UKHOGreg) [inline src/NCNEPortal/Startup.cs:132]: `services.AddRazorPages().AddRazorRuntimeCompilation();` is duplicated on Ln 70

**Review by UKHOGreg — COMMENTED** [review]
Just one duplicate line to remove I think,,,

File: src/NCNEPortal/Startup.cs (hunk 1 above)
Comment (tomling) [reply src/NCNEPortal/Startup.cs:132]: Good spot!

**Review by UKHOGreg — APPROVED** [review]

**Pull request merged**